superdesk/metadata/item.py

```diff
@@ -298,6 +298,11 @@
     'associations': {
         'type': 'dict',
     },
+    'alt_text': {
+        'type': 'string',
+        'nullable': True,
+        'mapping': not_analyzed,
```

Comment:
I don't know...
It was a suggestion from @akintolga, I think because we are already using `string` for `usageterms` and `copyrightnotice` and others...
Should we change it ?
what is the main difference ?